includes/output.inc

```diff
@@ -26,20 +26,15 @@ use Drush\Log\LogLevel;
  * @param $newline
  *    Add a "\n" to the end of the output.  Defaults to TRUE.
  */
-function drush_print($message = '', $indent = 0, $handle = NULL, $newline = TRUE) {
+function drush_print($message = '', $indent = 0, $handle = STDERR, $newline = TRUE) {
```

Comment:
I think `drush_print` should default to STDOUT. Switching to STDERR would be a backwards compatibility break.

Reply:
Opps, sorry, I didn't intend to do that. This is what happens when you can't just ```git pick``` because branches have changed!